src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -219,7 +219,7 @@ public function set($id, $service, $scope = self::SCOPE_CONTAINER)
      */
     public function has($id)
     {
-        for ($i = 2;;) {
+        for ($i = 2; ;) {
```

Comment:
I don't have a hard opinion on that one, but as we are using it that way everywhere, at least we are consistent. Having a space between the two ; looks weird to me.